components/ImageWithNounFallback.tsx

```diff
@@ -30,7 +28,8 @@ export function ImageWithNounFallback({
     )
   }
 
-  if (isDao) return <FallbackThumbnail tokenContract={tokenContract} tokenId={tokenId} />
+  // if (isDao)
+  return <FallbackThumbnail tokenContract={tokenContract} tokenId={tokenId} />
 
   return <></>
```

Comment:
intended?

Reply:
Oops, didn't save the file. Should remove isDao, which just checks if the collection address is in the old static list (LilNoun + Nouns only)

Because all of the data from the dynamic fetch is dao data, we don't need this check